package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -531,6 +535,11 @@ def _read_next_timestep(self, ts=None):
         except IndexError:
             raise IOError
 
+    def _get_dt(self):
```

Comment:
https://github.com/MDAnalysis/mdanalysis/blob/develop/package/MDAnalysis/coordinates/base.py#L706

`Timestep.dt` can ask the Reader when the trajectory only has time information